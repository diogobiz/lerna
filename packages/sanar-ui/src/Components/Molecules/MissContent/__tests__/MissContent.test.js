
import React from 'react'
import TestRenderer from 'react-test-renderer'

import ESMissContent from '../'

it('renders correctly', () => {
    const component = (
        <ESMissContent>
            <h1>ESMissContent</h1>
        </ESMissContent>
    )

    const tree = TestRenderer.create(component).toJSON()
    expect(tree).toMatchSnapshot()
})
