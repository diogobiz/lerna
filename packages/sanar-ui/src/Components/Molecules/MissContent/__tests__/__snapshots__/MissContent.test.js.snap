// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="es-miss-content"
>
  <img
    className="es-miss-content--img"
  />
  <div
    className="es-miss-content__content"
  >
    <div
      className="es-typography es-miss-content__content--text es-typography--regular es-typography__heading es-typography__heading--h5"
      data-testid="es-typography"
    />
  </div>
</div>
`;
